### ISSUE TITLE
Use of Keras Lambda

### ISSUE THREAD

**Issue by cgarciae** [lucidrains/lambda-networks#13]
Hey! Thank for the awesome implementations :D 

I was wondering why the use of `tf.keras.layers.Lambda`? Seems unnecessary, regular calls to TF operations works and is more readable. 

https://github.com/lucidrains/lambda-networks/blob/06a48f2a5b41f3cd278aee67838c32051a0a9bed/lambda_networks/tfkeras.py#L73

You can also call the functional version of the softmax instead.

**Comment by lucidrains** [comment]
@cgarciae no problem :) yeah, I'm not a keras expert, but I believe you are right have made all the changes you suggested
